tests/acceptance/features/bootstrap/NotificationContext.php

```diff
@@ -211,6 +340,24 @@ public function userShouldGetANotificationWithMessage(string $user, string $subj
 		);
 	}
 
+	/**
+	 * @Then user :user should not have a notification related to resource :resource with subject :subject
+	 *
+	 * @param string $user
+	 * @param string $resource
+	 * @param string $subject
+	 *
+	 * @return void
+	 */
+	public function userShouldNotHaveANotificationRelatedToResourceWithSubject(string $user, string $resource, string $subject):void {
+		$this->userListAllNotifications($user);
+		$this->featureContext->theHTTPStatusCodeShouldBe(200);
```

Comment:
why there is a `https-status` code check here and not in other places where `$this->userListAllNotifications($user);` is called

Reply:
i forget to remove this